app/components/dashboard/hypertension/hypertension_cascade_component.rb

```diff
@@ -0,0 +1,38 @@
+class Dashboard::Hypertension::HypertensionCascadeComponent < ApplicationComponent
+  attr_reader :data, :contactable, :period
+
+  FIXED_RATE_WHEN_NO_ESTIMATE = "2%" # We should a fixed rate when there's no esimate for the region
```

Comment:
[nit-pick]: Need to fix the typo in the comment.

Reply:
[question]: Does this mean  we need to show a 2% rate when there is no estimated population for a district? Looks like it's not adding that fixed rate to the card.
<img width="451" alt="image" src="https://github.com/simpledotorg/simple-server/assets/32141642/fcc2a75e-d8de-4d89-aa30-f6440a49a4a5">

Reply:
We don't need to show the actual percentage, since there's no denominator to get the %. We just set the graphs to 2% for a visual cue that nothing compares.

Reply:
Gotcha!